File: Concurs Inot C#/IRepoProba.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Problema_3_CSharp.me.model;
namespace Problema_3_CSharp.me.repository
{
    public interface IRepoProba : IRepository<int,Proba>
    {
        IEnumerable<Proba> FindProbeDupaStil(Stiluri stil);
        IEnumerable<Proba> FindProbeDupaDistanta(float distanta);
    }
}

File: IRepoProba.java
package me.repository;

import me.model.Proba;
import me.model.Stiluri;

import java.util.List;

public interface IRepoProba extends Repository<Integer, Proba> {
    List<Proba> findProbeDupaStil(Stiluri stil);
    List<Proba> findProbeDupaDistanta(float distanta);
}
